docs/docs/authoring.md

```diff
@@ -12,29 +12,89 @@ This guide specifically applies to .md files which will be converted to the Prai
 
 ### Authoring Questions Details
 
-- [Video 1 - Overview of Creating PL problems](https://www.youtube.com/watch?v=hFYwxmLDORw&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wamz)]
+- [Video 1 - Overview of Creating PL problems](https://www.youtube.com/watch?v=hFYwxmLDORw&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wamz)
 - [Video 2 - Randomizing and solving physics problem in Python](https://www.youtube.com/watch?v=CCnc7bspuZg&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wam&index=2)
 - [Video 3 - Writing a Markdown file for a numerical-entry problem](https://www.youtube.com/watch?v=ZoZYZlmvh_Y&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wam&index=3)
 - [Video 4 - Fully implemented numeric entry problem](https://www.youtube.com/watch?v=ujCbACEbizA&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wam&index=4)
 - [Video 5 - Wrting a symbolic PL problem](https://www.youtube.com/watch?v=P0sK1WS2p98&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wam&index=5)
 - [Video 6 - Making a multi-part problem](https://www.youtube.com/watch?v=ftPXBcMCROc&list=PLfhjdV-pwMOa7HeYtI4Qd9QRMPfv77Wam&index=6)
 
-## Step by step flow for authoring questions
+## Overview of the steps for authoring questions
 
-More details about this are laid out [here](authoring_questions).
+This is an overview of the steps you need to follow to author each question once you have completed the initial setup.
 
-1. Clone the GitHub repository locally (`git clone <clone_url>`).
-2. Go into the `instructor_physics_bank` directory using `cd`.
-3. Make a new branch for your question with an appropriate name (`git checkout -b <new_branch_name>`).
-4. Create a new directory in the `source/YourTopic/QuestionTitle` or `source/YourTopic/YourSubtopic/QuestionTitle` directory to store your markdown file and any assets (usually images).
-5. Create your markdown file and copy the format from one of the [templates](https://github.com/open-resources/instructor_physics_bank/tree/main/templates).
-6. Edit the markdown file to author the question.
-7. Make your `add, commit and push` commands to push to the GitHub repository.
-8. Create a `pull request` and add two reviewers to review your work.
-9. Add the `syntax check` label on GitHub and make sure it passes, fix any errors if it does not.
-10. If the syntax check passes, go on [PrairieLearn](https://ca.prairielearn.org/pl/course_instance/2320/instructor/course_admin/questions) and check if it looks as desired.
-11. Add the `ready_to_merge` label to mark for final review.
-12. Perform `git pull` for any changes and go back to step 3 for another question.
+1. Make a `new branch` for each new question.
+2. Create a `new directory` in the source folder under your topic or subtopic.
+3. Crate your `markdown file` in the folder and add any `assets` such as images being used for your question.
+4. Copy the markdown `template` according to your question and start editing.
+5. `Add`, `commit` and then `push` to the Github repository.
+6. Create a pull request and add 2 reviewers to approve your work.
```

Comment:
I think we've gone down to 1 reviewer now, but maybe we want 2 for new people?

Reply:
We can say "1 or 2 reviewers" at this point.

Reply:
Okay I will make that change